src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,20 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-size: 12px;
+
+  line-height: 14px;
+}
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
It's not `reset` styles, rather `normalize`. Move these styles to normalize.scss